core/model_induction.py

```diff
@@ -170,12 +170,23 @@ def dump(self, file_path):
             file.write(file_buffer)
 
 
-def train_decision_tree(train_features, train_label):
+def train_custom_tree(train_features, train_label):
     model = CustomBinaryClassifier()
     model.fit(train_features, train_label)
     return model
 
-def train_classifier_tree(train_features, train_label):
-    model = RandomForestClassifier(n_estimators=30, max_depth=40)
-    model.fit(*balance_binary_dataset(train_features, train_label, skew_false=8))
+def train_decision_tree(train_features, train_label):
+    model = DecisionTreeClassifier()
+    model.fit(train_features, train_label)
+    return model
+
+def train_classifier_tree(train_features, train_label, **kwargs):
```

Comment:
i see what you're trying to do with the names here but these are all classifiers so this feels like a misnomer 🧐

Reply:
Indeed 🧐


Will update